docs/infrastructure/index.asciidoc

```diff
@@ -23,9 +23,7 @@ instructions and information.
 [float]
 == Configure data sources 
 
-By default the Infrastructure UI uses the `metricbeat-*` index pattern to query the data.
-If you configured Metricbeat to export data to a different set of indices you will need to set `
-xpack.infra.sources.default.metricAlias` in `config/kibana.yml` to match your index pattern. You can also configure the timestamp field by overriding `xpack.infra.sources.default.timestamp`.
+By default the Infrastructure UI uses the `metricbeat-*` index pattern to query the data. If you configured Metricbeat to export data to a different set of indices, you will need to set `xpack.infra.sources.default.metricAlias` in `config/kibana.yml` to match your index pattern. You can also configure the timestamp field by overriding `xpack.infra.sources.default.fields.timestamp`.See {infrastructure-ui-settings}[Infrastructure Ui Settings] for a complete list.
```

Comment:
Use `<<infrastructure-ui-settings-kb>>` instead of `{infrastructure-ui-settings}[Infrastructure Ui Settings]` here. The other syntax exists for links to other books, not internal links.  In fact, I don't see `{infrastructure-ui-settings}` defined anywhere, so attribute resolution fails (silently) when I build this. 

Also add a space between the period and "See"

Reply:
I honestly don't know what I'm doing with regards to linking OR even how ASCII doc works beyond editing. How does one go about defining `{infrastructure-ui-settings-kb}`?

Reply:
You don't need to do that here because the topic you're pointing to is in the same book. So use the internal links syntax: `<<infrastructure-ui-settings-kb>>`

You can read all IDs and linking [here](https://github.com/elastic/docs/blob/master/README.asciidoc#linking).